commands/channel_test.go

```diff
@@ -1236,33 +1222,33 @@ func (s *MmctlUnitTestSuite) TestAddChannelUsersCmdF() {
 		s.Len(printer.GetLines(), 0)
 		s.Len(printer.GetErrorLines(), 0)
 	})
-	s.Run("Add existing user to existing channel", func() {
+	s.Run("Add existing user user@example.coto existing channel", func() {
```

Comment:
I think this change comes from a bad paste

```suggestion
	s.Run("Add existing user to existing channel", func() {
```